apps/mdn/mdn-aws/k8s/Makefile

```diff
@@ -368,13 +388,17 @@ k8s-api:
 		KUMA_CPU_REQUEST=${API_CPU_REQUEST} \
 		KUMA_MEMORY_REQUEST=${API_MEMORY_REQUEST} \
 		KUMA_ALLOWED_HOSTS=${API_ALLOWED_HOSTS} \
+		NEW_RELIC_APP_NAME=${NEW_RELIC_API_NAME} \
+		NEW_RELIC_MONITOR_MODE=${NEW_RELIC_MONITOR_MODE} \
+		NEW_RELIC_CONFIG_FILE=${NEW_RELIC_CONFIG_FILE} \
```

Comment:
I think these two lines:
```
NEW_RELIC_MONITOR_MODE=${NEW_RELIC_MONITOR_MODE} \
NEW_RELIC_CONFIG_FILE=${NEW_RELIC_CONFIG_FILE} \
```
are not needed since they will be picked up from the existing exported env variables?